src/Symfony/Component/Form/Extension/Core/DataTransformer/NumberToLocalizedStringTransformer.php

```diff
@@ -93,13 +93,16 @@ class NumberToLocalizedStringTransformer implements DataTransformerInterface
      */
     const ROUND_HALFDOWN = \NumberFormatter::ROUND_HALFDOWN;
 
+    /*
```

Comment:
One last comment sorry :) `/**` here?
But more importantly (for @stof or @Tobion): shouldn't we also deprecate the $grouping and $roundingMode props and make the private in 3.0?